server/server.js

```diff
@@ -1,5 +1,14 @@
-const a = 10;
+const express = require('express');
+const parser = require('body-parser');
+const path = require('path');
+const axios = require('axios');
```

Comment:
What are you planning on using axios in your server for? Any external calls to other apis?